code/modules/atmospherics/gasmixtures/immutable_mixtures.dm

```diff
@@ -43,6 +43,11 @@
 	. = ..()
 	temperature = initial_temperature
 
+/datum/gas_mixture/temperature_share(datum/gas_mixture/immutable/sharer, conduction_coefficient, sharer_temperature, sharer_heat_capacity)
```

Comment:
```
/datum/gas_mixture/proc/temperature_share(datum/gas_mixture/sharer, conduction_coefficient)
	//Performs temperature sharing calculations (via conduction) between two gas_mixtures assuming only 1 boundary length
	//Returns: new temperature of the sharer
```
In https://github.com/tgstation/tgstation/blob/master/code/modules/atmospherics/gasmixtures/gas_mixture.dm.

I have no idea if this work but I sure know this is hacky.

Reply:
Oh. Also this fucking runtimes if the sharer ISN'T datum/gas_mixture/immutable

Reply:
indeed it does, back to the drawing board